services/ymax-planner/test/mocks.ts

```diff
@@ -76,10 +84,24 @@ export const createMockProvider = () => {
         listeners.forEach(listener => listener(log));
       }
     },
-    waitForBlock: blockTag => {},
+    waitForBlock: _blockTag => {},
     getBlockNumber: async () => {
       return currentBlock;
     },
+    getBlock: async (blockNumber: number) => {
+      const blocksAgo = latestBlock - blockNumber;
+      const ts = currentTimeMs - blocksAgo * avgBlockTimeMs;
+      return { number: blockNumber, timestamp: Math.floor(ts / 1000) };
+    },
+    getLogs: async (args: { fromBlock: number; toBlock: number }) => {
+      if (events === undefined)
+        throw new Error('No event data provided in mock');
```

Comment:
Nit: We should always use braces with multi-line if/else/etc. statements (but this one can be single-line).
```suggestion
      if (events === undefined) throw Error('No event data provided in mock');
```